### ISSUE TITLE
Time only 12 hour formatting - MazPicker component

### ISSUE THREAD

**Issue by emg36** [LouisMazel/maz-ui#348]
Only time doesn't seem to format to 12 hour

```
<maz-picker
    v-model="startTime"
    placeholder="Start time"
    :format="'hh:mm a"
    onlyTime
>
```
startTime = '8:02 pm'

The startTime variable saves is as I would expect but the input field will show 20:02 where I would have thought that it would show the same format.

I have tried passing timeStyle e.g.

```
:inputDateStyle="{
    dateStyle: 'short',
    timeStyle: 'short'
}
```

but it doesn't seem to change the input value.

![CleanShot 2022-09-19 at 20 43 48](https://user-images.githubusercontent.com/5077135/190980781-ce23c25e-4bbc-4cd1-bba8-7d9412771d50.png)

**Comment by LouisMazel** [comment]
Hi @emg36 

You made a mistake in the value of the format prop. Delete the `'` in it. 

Please use this code instead : 

```html
<MazPicker
    v-model="startTime"
    placeholder="Start time"
    :format="hh:mm a"
    only-time
/>
```

**Comment by emg36** [comment]
Hi @LouisMazel

That was a mistake writing out the issue sorry. It still does behaves the same.

**Comment by LouisMazel** [comment]
This issue is solved in v3.0.3

Thanks for your report @emg36 ! 🔥
